community/modules/scripts/spack-install/README.md

```diff
@@ -48,23 +48,49 @@ see this module used in a full blueprint, see the [spack-gromacs.yaml] example.
         scope: defaults
         value: |
           modules:
-            tcl:
-              hash_length: 0
-              all:
-                conflict:
-                  - '{name}'
-              projections:
-                all: '{name}/{version}-{compiler.name}-{compiler.version}'
+            default:
+              tcl:
+                hash_length: 0
+                all:
+                  conflict:
+                    - '{name}'
+                projections:
+                  all: '{name}/{version}-{compiler.name}-{compiler.version}'
       compilers:
       - gcc@10.3.0 target=x86_64
       packages:
       - cmake%gcc@10.3.0 target=x86_64
       environments:
       - name: main-env
+        type: packages
         packages:
         - intel-mkl%gcc@10.3.0 target=skylake
         - intel-mpi@2018.4.274%gcc@10.3.0 target=skylake
         - fftw%intel@18.0.5 target=skylake ^intel-mpi@2018.4.274%intel@18.0.5 target=x86_64
+      - name: explicit-env
+        type: file
+        value: |
```

Comment:
Just an idea, let me know what you think:
We could have an environments directory in the spack module and have this "explicit env" point to that. That way, we can have the files run through the yaml linter and decrease the size of this example blueprint. We'd also be able to grow our example environments more naturally, and users could swap or add only a few lines in the example to build a different stack.

That would be a follow up of course, no action needed here.

Reply:
Sure, so in this case you mean have something like an external-file option, or something. That would point to an external environment definition, right? I think that could be a good idea, and shouldn't be too hard to add next.

Reply:
since this is discussing future work, we can track it separately.